packages/components/MIGRATING.md

```diff
@@ -66,6 +69,105 @@ Changes:
 </Label>
 ```
 
+### Dialog
+
+Changes:
+
+-   The `Dialog` component now controls its open state internally by default.
+
+    -   You can pass in a trigger component as a child of `Dialog.Trigger` to control the open state.
+
+    -   The `open` and `onOpenChange` props can still be used to control the open state externally if needed.
+
+-   The `Dialog` now provides multiple subcomponents.
+
+    -   _required_ - The `Dialog.Root` wraps all other subcomponents and handles the dialog state and modality.
```

Comment:
can you document `modality` and what is the new prop name / value

Reply:
added